server/src/project/project.service.ts

```diff
@@ -283,8 +284,50 @@ export class ProjectService {
   constructor(
     private readonly dynamicsWebApi: OdataService,
     private readonly config: ConfigService,
+    private readonly carto: CartoService,
   ) {}
 
+  async getBblsGeoJSON(bbls) {
```

Comment:
Maybe I should rename as "getBblsGeometry"?

Reply:
That works too